dev/com.ibm.ws.jpa.tests.jpa_32_fat/bnd.bnd

```diff
@@ -0,0 +1,48 @@
+#*******************************************************************************
+# Copyright (c) 2022,2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src,\
+    test-applications/helpers/DatabaseManagement/src,\
+    test-applications/jpabootstrap/src
+
+fat.project: true
+fat.test.databases: true
+tested.features: el-3.0, persistence-3.0, servlet-5.0, cdi-3.0, beanValidation-3.0, enterprisebeanslite-4.0, persistence-3.1, persistencecontainer-3.1, xmlbinding-4.0, servlet-6.0, persistence-3.2, persistencecontainer-3.2, servlet-6.1
```

Comment:
This does not test this older feature levels.... remove everything up through servlet-6.0.  Just the last 3 should remain.

Reply:
Done